pyomo/pysp/scenariotree/__init__.py

```diff
@@ -9,9 +9,37 @@
 #  ___________________________________________________________________________
 
 import pyomo.pysp.scenariotree.util
-from pyomo.pysp.scenariotree.tree_structure_model import *
-from pyomo.pysp.scenariotree.tree_structure import *
-from pyomo.pysp.scenariotree.instance_factory import *
+from pyomo.pysp.scenariotree.tree_structure_model import (
+    CreateAbstractScenarioTreeModel, CreateConcreteTwoStageScenarioTreeModel,
+    ScenarioTreeModelFromNetworkX)
+from pyomo.pysp.scenariotree.tree_structure import (ComponentMap, value,
+                                                    minimize, maximize, Var,
+                                                    Expression, Block,
+                                                    Objective, SOSConstraint,
```

Comment:
Same question here. Should some of these imports come from `pyomo.core` or `pyomo.environ` instead?

Reply:
I incorporated all of your suggestions. Let's see if the tests pass...